eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubConsumerOptions.java

```diff
@@ -193,55 +160,37 @@ public Retry retry() {
     }
 
     /**
-     * Gets the priority for this receiver. If {@link Optional#isPresent()} is {@code false}, then this is not an
-     * exclusive receiver. Otherwise, it is and there can only be one receiver per (partition + consumer group)
-     * combination.
+     * Gets the priority for this consumer. If {@link Optional#isPresent()} is {@code false}, then this is not an
```

Comment:
nit:  Probably want to massage the `priority` nomenclature.   The level is an implied priority, but this reads pre-rename to me.